src/routes/firstRouter.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import * as firstController from '../controllers/firstController';
```

Comment:
as for me it is better to destructure function like that
import { getRoute1, postRoute1, deleteRoute1 } from '../controllers/firstController';
that allow you to understand what function you use from first line on function